modules/plot/main/coffee/Utils.coffee

```diff
@@ -1,3 +1,19 @@
+doCollisionDetection = (nodes) ->
+  distance = 1
+  nodes.forEach (node, i) ->
+    previousNodes = nodes.slice 0, (i - distance + 1)
+    doesOverlap = previousNodes.find (previousNode, j) ->
+      prevBox = previousNode.getBoundingClientRect()
+      currBox = node.getBoundingClientRect()
+      doesCollide = currBox.left < prevBox.right
+      currentDistance = i - j
```

Comment:
Could we name `i` and `j` a bit better? (nodeIndex and previousIndex or something along those lines?)